### DISCUSSION TITLE
RubyCache debug not generating output

### DISCUSSION THREAD

**Discussion by Krishnan1714** [gem5/gem5#1359] [Q&A]
Hi all. I am trying to trace gem5 rubycache. I am running a 4x4 mesh topology with XY routing with mesi two level. I am unable to generate any debug with RubyHitMiss, RubyGenerated, RubyStats, RubySequencer. RubySlicc Debug produces mainly "not present" states, rest of the information is unavailable. Benchmark for single cpu with single directory produces result,problem is for multi-cpus.Only Ruby, RubyCache Debugs work. It was working perfectly for me earlier. I have tried making a clean X86 build, still its not working. Below are my build and run commands

`scons build/X86/gem5.opt RUBY=True PROTOCOL=MESI_Two_Level -j12`
`build/X86/gem5.opt --debug-flags=RubyHitMiss configs/deprecated/example/se.py --num-cpus=16 --num-dirs=16 --sys-clock=2GHz --topology=Mesh_XY --mesh-rows=4 --ruby --num-l2caches=16 --network=garnet --mem-size=8GB --routing-algorithm=1 --router-latency=3 -I 1000 --fast-forward=100000 --bench=namd-namd-namd-namd-namd-namd-namd-namd-namd-namd-namd-namd-namd-namd-namd-namd --cpu-type=O3CPU`

Kindly help

**Comment by Adarshssuresh** [comment]
Try reducing fastforwarding . It may help
